Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM ubuntu:disco
+
+# Install packages required to build AsteroidOS
+RUN apt update && apt upgrade -y && apt install -y git build-essential cpio diffstat gawk chrpath texinfo python2 python3 wget shared-mime-info
+
+# Add the en_US.utf8 locale because it is required and not installed by defualt in minimal Debian images
```

Comment:
Could you fix the "by defualt" -> "by default"
You also mention "minimal Debian images" while the container is based on Ubuntu

Reply:
Yes, I updated it. :)